jupyter_events/schema.py

```diff
@@ -144,6 +144,10 @@ def version(self) -> int:
         """Schema's version."""
         return self._schema["version"]
 
+    @property
+    def properties(self) -> dict:
+        return self._schema["properties"]
```

Comment:
Will this include nested properties?  Like if `"message"` only resides on an object-valued property's properties list?

Reply:
Added a unit test to make sure that nested "message" fields are emitted.